uportal-war/src/main/java/net/sf/ehcache/distribution/jgroups/BootstrapRequest.java

```diff
@@ -0,0 +1,161 @@
+/**
+ *  Copyright 2003-2010 Terracotta, Inc.
```

Comment:
Copyright Terracotta? Is this covered by one of the ehcache mentions in `NOTICE` or need further acknowledgement?